shared/login/signup/index.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+
+import React from 'react'
+import Form from '../form'
+import ErrorText from '../error.render'
+
+// Signup Components
+import InviteCode from './inviteCode'
+import UsernameEmailForm from './usernameEmailForm'
+import PassphraseSignup from './passphrase'
+import DeviceName from './deviceName'
+
+import {Map} from 'Immutable'
+import type {URI} from '../../reducers/router'
+
+export default function SignupRouter (currentPath: Map<string, string>, uri: URI): any {
```

Comment:
You're right, should not be capitalized. This is a normal function not a factory or class